test/Tools/docGeneratorTestFiles/TestMdOutput_CoreNodeModels/CoreNodeModels.Remember.md

```diff
@@ -0,0 +1,4 @@
+## Remember - Documentation
+This documentation file is auto generated by NodeDocumentationMarkdownGenerator, Version=2.13.0.2212, Culture=neutral, PublicKeyToken=null.
```

Comment:
this seems like a great node to supply better documentation for.
FYI @hwahlstrom